components/accessibility-helpers/src/SrSkipLink.js

```diff
@@ -2,51 +2,48 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { jsx, useTheme } from '@westpac/core';
+import { jsx } from '@westpac/core';
 
 // ==============================
 // Component
 // ==============================
 
-export const SrSkipLink = ({ ...props }) => {
-	const {
-		typography: { link },
-	} = useTheme();
+export const SrSkipLink = props => (
+	<a
+		css={{
+			position: 'absolute',
+			width: 1,
+			height: 1,
+			margin: -1,
+			padding: 0,
+			overflow: 'hidden',
+			clip: 'rect(0,0,0,0)',
+			border: 0,
+			fontSize: '1.3125rem',
 
-	const common = {
-		position: 'absolute',
-		width: 1,
-		height: 1,
-		margin: -1,
-		padding: 0,
-		overflow: 'hidden',
-		clip: 'rect(0,0,0,0)',
-		border: 0,
-		fontSize: '2.1rem',
+			':active, :focus': {
+				position: 'fixed',
+				top: 0,
+				right: 0,
+				left: 0,
+				height: 'auto',
+				width: 'auto',
+				margin: 0,
+				padding: '1em',
+				overflow: 'visible',
+				clip: 'auto',
+				textAlign: 'center',
+				zIndex: 2000,
+				backgroundColor: '#fff',
+			},
 
-		':active, :focus': {
-			position: 'fixed',
-			top: 0,
-			right: 0,
-			left: 0,
-			height: 'auto',
-			width: 'auto',
-			margin: 0,
-			padding: '2.1rem',
-			overflow: 'visible',
-			clip: 'auto',
-			textAlign: 'center',
-			zIndex: 2000,
-			backgroundColor: '#fff',
-		},
-
-		':focus': {
-			outlineOffset: -link.focus.outlineOffset, //override to be inside
-		},
-	};
-
-	return <a css={common} {...props} />;
-};
+			':focus': {
+				outlineOffset: -2, //override to be inside
```

Comment:
This offset override depends on the focus outline width setting (defined in Core). 

Should this somehow link to that setting in Core? Or maintain this setting separately and hope they always align?